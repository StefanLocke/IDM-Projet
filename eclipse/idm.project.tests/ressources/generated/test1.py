import pandas as pd
df = pd.read_csv( "C:/Users/alexl/Documents/FAC/M2/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/input/fruits.csv" )
df.to_csv("C:/Users/alexl/Documents/FAC/M2/IDM/IDM-Projet/eclipse/idm.project.tests/ressources/output/test1Python.csv", index=False)

