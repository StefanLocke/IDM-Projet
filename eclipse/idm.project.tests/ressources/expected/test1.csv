,Article,Stock,Prix,Benefice
0,Pomme,2,1,2
1,Banane,10,2,20
2,Peche,5,6,30
3,Croissant,15,18,2
4,Coca Cola,100,16,4
5,Sandwich,18,11,7
